index.js

```diff
@@ -1,12 +1,129 @@
 'use strict';
 
-const xd = 'test';
-
-/*
-1. shuffle deck of cards
-2. deal the cards between 2 players
-3. play the game: higher value wins, lower value loses, draw = no points for any player
-4. display the players score and result of each round
-5. display who won the game
-6. test the whole thing using unit tests
- */
+const fs = require('fs');
+
+/* - - - - -  T A S K S / R U L E S - - - - -
+1. Create a deck of cards and shuffle it ✓
+2. Deal the cards between 2 players (Half-Half or One-By-One) ✓
+3. Play the game: higher card value wins, lower loses, draw = no points for any player ✓
+4. Display the players score and result of each round ✓
+5. When hand is empty display who won the game ✓
+6. Test the whole thing using unit tests ✗
+*/
+
+// Players:
+const p1 = { startDeck: [], score: 0, poolDeck: [] };
+const p2 = { startDeck: [], score: 0, poolDeck: [] };
+
+// Reads the whole deck from the file
+const path = 'test/cardDeck.txt';
+const wholeDeck = fs.readFileSync(path, 'utf8')
+  .trim()
+  .split('\n')
+  .map(line => line.split(', '));
+
+// Shuffles any array (Fisher–Yates shuffle)
```

Comment:
very good, but do you understand how this algorithm works?

Reply:
yup, but used .sort to shuffle instead